src/main/java/net/starschema/clouddb/jdbc/BQSupportFuncts.java

```diff
@@ -81,6 +81,8 @@ public static String constructUrlFromPropertiesFile(Properties properties, boole
         dataset = dataset == null ? properties.getProperty("dataset") : dataset;
 
         String forreturn = "";
+        // Set to '?' for the first param and '&' for subsequent params.
+        String paramSep = "?";
```

Comment:
Unfortunately there doesn't seem to be a standard URL builder for java. I agree something a little more elegant than this would be ideal, but this is pretty simple and easy to understand which is also a plus.